docs/source/release_notes.rst

```diff
@@ -14,6 +14,7 @@ Release Notes
         * Fixed bug where ``score_pipelines`` method of ``AutoMLSearch`` would not work for time series problems :pr:`2786`
     * Changes
         * Deleted ``EmptyDataChecks`` class :pr:`2794`
+        * Removed data check for log distributions in ``make_pipeline`` :pr:`2806`
```

Comment:
I think this is technically a breaking change.

Reply:
I always think of a breaking change as something that would cause our API to change, not the implementation, but I'm down to add it so that we're loud about the behavior of AutoML changing 😁